### PR TITLE
Feature/backend configuration

### LINKED ISSUE #14: Backend configuration
Implement nconf-based configuration for the backend application

### PR DESCRIPTION
**Pull request by R1ZZU**
Closes #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,10 @@
   },
   "scripts": {
     "prepush": "npm test",
-    "test": "node ./node_modules/karma/bin/karma start ./karma.conf.coffee --single-run"
+    "test": "node ./node_modules/karma/bin/karma start ./karma.conf.coffee --single-run",
+    "start": "NODE_PATH=./app coffee service.coffee",
+    "dev": "NODE_ENV=dev npm start",
+    "debug": "node-inspector 2>/dev/null && NODE_PATH=./app NODE_ENV=dev coffee --nodejs --debug-brk service.coffee"
```
Comment (DaQuirm) [inline package.json]: kudos for that :smile_cat:

File: service.coffee [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cssqdConfig = require 'cssqd-config'
+path        = require 'path'
+
+cssqdConfig
+	.env
+		separator: '__'
```
Comment (DaQuirm) [inline service.coffee]: What is this `separator` thing for?

Comment (R1ZZU) [inline service.coffee]: cssqdConfig.get('some:nested:variable') will read some__nested__variable from env

Comment (DaQuirm) [inline service.coffee]: Ah, I see :cat: 
But why two underscores then?

Comment (R1ZZU) [inline service.coffee]: to differ nesting and :snake:_case

Comment (DaQuirm) [inline service.coffee]: Got it :smiley_cat:

**Comment by DaQuirm** [comment]
LGTM

**Comment by R1ZZU** [comment]
Great!
@AndreyLukashenko, @kirilknysh, take a look please

**Comment by AndreyLukashenko** [comment]
LGTM

**Pull request merged**